csrankings-v.csv

```diff
@@ -410,6 +410,7 @@ Vlad Rusu,CRIStAL,https://www.cristal.univ-lille.fr/profil/rusu,uDBCQR8AAAAJ
 Vladik Kreinovich,University of Texas - El Paso,http://hb2504.utep.edu/vitas/vladik.pdf,iern2agAAAAJ
 Vladimir A. Kulyukin,Utah State University,https://cs.usu.edu/people/faculty/kulyukin-vladimir,NOSCHOLARPAGE
 Vladimir Braverman,Rice University,https://profiles.rice.edu/faculty/vladimir-braverman,DTthB48AAAAJ
+Vladimir Braverman,Johns Hopkins University,https://www.cs.jhu.edu/~vova/,DTthB48AAAAJ

```

Comment:
<img width="1235" alt="Screen Shot 2024-08-05 at 10 05 54 AM" src="https://github.com/user-attachments/assets/b4149077-192d-40c8-8c4f-42afd0161f18">

From this page: https://www.cs.jhu.edu/faculty/